client/src/main/java/gui/controllers/PlacementSceneController.java

```diff
@@ -61,9 +60,13 @@ public void initialize() {
 
     @FXML
     private void placeRandom() {
+        appInstance.getServerConnection().sendMessage("ASK_FOR_FLEET");
```

Comment:
Some enum instead of literal "ASK_FOR_FLEET" ?

Reply:
Done with private static final.